configure.ac

```diff
@@ -861,7 +861,11 @@ if test "$use_hardening" != "no"; then
   esac
 
   case $host in
-    *aarch64*)
+    *aarch64-*-linux*)
+      AX_CHECK_COMPILE_FLAG([-mbranch-protection=standard], [HARDENED_CXXFLAGS="$HARDENED_CXXFLAGS -mbranch-protection=standard"])
```

Comment:
Will this override the user if he sets a stronger mode?

Reply:
The same logic as all other flags applies, the user has the final say.